src/AlertBox/AlertBox.tsx

```diff
@@ -1,6 +1,6 @@
 import _ from "lodash";
 import classnames from "classnames";
-import React, {PureComponent, PropTypes} from "react";
+import * as React from "react";
```

Comment:
Ah right, I actually couldn't do `import * as classnames` because of a limitation of Rollup/ES6
 Modules—tools like Webpack and TypeScript, for backwards compatibility with old node-style exports, interprets `import *` calls for non-ES6-module code as a way to access the `module.exports` variable. But in pure ES6 semantics, `import *` actually returns a namespace object, which is not invocable. So, Rollup complains.

It works here because I set in `tsconfig.json` to allow the synthetic module imports, so that `import classnames` gets interpreted as importing `module.exports`; however, `React` actually is being exported properly, so it uses actual ES6 semantics, which calls `import React` an incorrect usage.

This is further complicated since Babel is even more lenient. It will all you to refer to default exports via `import classnames` as though `module.exports` were a default export, whereas `TypeScript` requires the synthetic imports flag to make that happen.

https://github.com/rollup/rollup/issues/670